src/users/src/users-service/routes.go

```diff
@@ -47,6 +49,30 @@ var routes = Routes{
 		"/users/identityid/{identityID}",
 		UserShowByIdentityId,
 	},
+	Route{
+		"GetFilteredUser",
+		"GET",
+		"/users/filter/",
```

Comment:
Can we change this to `GET /users` or `GET /users/all` with optional query string parameters that can be used to filter based on primary persona, age range, etc?

Reply:
I updated the endpoint to users/unclaimed with params of ageRange,primaryPersona and count.
I'd like to keep this as is and not merge it into users/ users/all since this API is built to only return users for the shopper-select criteria for a very specific use.
The rest of the comments on the PR, I agree with and have made updates.